scripts/build/builders/ameba.py

```diff
@@ -21,6 +21,7 @@
 
 class AmebaBoard(Enum):
     AMEBAD = auto()
+    LIGHT = auto()
```

Comment:
This should probably be in AmebaApp not board.

CI fails because of this.

Reply:
Oh right, that was a typo, thanks for pointing it out!